frontend/src/Settings/Indexers/Indexers/EditIndexerModalContent.js

```diff
@@ -153,6 +154,23 @@ function EditIndexerModalContent(props) {
                 />
               </FormGroup>
 
+              <FormGroup
+                advancedSettings={advancedSettings}
+                isAdvanced={true}
+              >
+                <FormLabel>Maximum Single Episode Age</FormLabel>
+
+                <FormInputGroup
+                  type={inputTypes.NUMBER}
+                  name="maximumSingleEpisodeAge"
+                  helpText="During search, only season packs will be allowed when the season's most recent episode is older than this setting. Standard series only. Use 0 to ignore."
```

Comment:
I think `Use 0 to disable.` reads a bit better.

Reply:
Since this explicitly only triggers when a search for a season (or at least not a single episode only) is performed I think that should be more explicit.

Reply:
How about:
"During full season search, only season packs will be allowed when the season's most recent episode is older than this setting. Standard series only. Use 0 to disable."

Reply:
Just a small tweak, but I think either:

`During a full season search...`
or
`During full season searches`

works a bit better.

Reply:
Done